.github/workflows/lint.yml

```diff
@@ -0,0 +1,23 @@
+name: Lint
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9.1
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements_dev.txt
+      - name: Lint with flake8
+        run: flake8
+      - name: Run black
+        if: ${{ always() }}
+        uses: psf/black@stable
+        with:
+          args: --line-length 79 --check
```

Comment:
Could we use black's default for this? Its what we use elsewhere, and I find the slightly longer lines quite a bit more readable.

Reply:
then it reformats all the lines to be longer and flake8 fails. We can up the line length for flake8 instead? I'm not fussed, though the gh PR review ui will wrap them